### PR TITLE
Gens cancer button

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
My current understanding is population changed a little with #4109, cf the controller loop that does only the case keys before vs all evaluation categories in the next. 
The error happens in the small pinned summary table, when the `acmg_classification` does not exist at all, or is broken for reasons. The latter, I still do not understand. But, here is the obvious fix for the template.
I would still be grateful for a dig into particularily what might be other reasons for the acmg_classification case key to be missing although classifications do seem to exist. Presumably some weird combo of add and remove.

File: scout/server/blueprints/cases/templates/cases/individuals_table.html [hunk 1]
```diff
@@ -28,7 +28,9 @@
         <tbody>
           {% for ind in case.individuals %}
             <tr {% if ind.phenotype_human == 'tumor' %} class="bg-danger-light" {% endif %}>
-              <td>{{ ind.display_name }}</td>
+              <td>{{ ind.display_name }}{% if gens_info.display %}
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/individuals_table.html:31]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
Looks good to me!

**Review by mathiasbio — COMMENTED** [review]
Hi!

I'm sorry I should probably have been more specific in my request of this CNprofile button. GENS will only be active for WGS cases, and I wonder if it would make sense to only include the CNprofile button for WGS cases as for TGA it would be a dead button?

Thanks!

**Comment by dnil** [comment]
Ok, the gens for Balsamic WGS is on track for a launch on Friday, so lets enable this, but only for WGS cases as discussed.

**Pull request merged**